src/Shared/LoggingExtensions.cs

```diff
@@ -6,9 +6,30 @@ namespace Workleap.DomainEventPropagation;
 // https://docs.microsoft.com/en-us/dotnet/core/extensions/logger-message-generator
 internal static partial class LoggingExtensions
 {
-    [LoggerMessage(1, LogLevel.Information, "The domain event type {DomainEventName} could not be found for event with subject {Subject}")]
+    [LoggerMessage(1, LogLevel.Information, "The cloud event with subject {Subject} could not be wrapped into Domain Event")]
+    public static partial void CloudEventNotSerializedToWrapper(this ILogger logger, string subject);
+    
+    [LoggerMessage(2, LogLevel.Information, "The domain event type {DomainEventName} could not be found for event with subject {Subject}")]
     public static partial void EventDomainTypeNotRegistered(this ILogger logger, string domainEventName, string subject);
 
-    [LoggerMessage(2, LogLevel.Information, "A handler for the domain event {DomainEventName} was not registered")]
+    [LoggerMessage(3, LogLevel.Information, "A handler for the domain event {DomainEventName} was not registered")]
     public static partial void EventDomainHandlerNotRegistered(this ILogger logger, string domainEventName);
+    
+    [LoggerMessage(4, LogLevel.Warning, "The event with {eventId} {EventName} will be rejected.")]
```

Comment:
Keep it Information, c'est un expected path, ça va juste spam le monde en warning